### PR TITLE
[CoreRT armel] fix for PINVOKE calls in CoreRT

### PR DESCRIPTION
**Pull request by sergign60**

### CODE REVIEW

**Comment by sergign60** [comment]
CC: @jkotas @Dmitri-Botcharnikov @BredPet @mskvortsov

**Comment by sergign60** [comment]
@jkotas please review

File: src/jit/codegencommon.cpp [hunk 1]
```diff
@@ -8687,8 +8687,10 @@ void CodeGen::genFnProlog()
     {
         excludeMask |= RBM_PINVOKE_FRAME;
 
+#ifndef _ARM_
         assert((!compiler->opts.ShouldUsePInvokeHelpers()) || (compiler->info.compLvFrameListRoot == BAD_VAR_NUM));
```
Comment (jkotas) [inline src/jit/codegencommon.cpp]: This does not look right. `compLvFrameListRoot` should not be allocated at all with ShouldUsePInvokeHelpers.

Comment (sergign60) [inline src/jit/codegencommon.cpp]: @kotas Yes, you're right. I've made the mistake

**Pull request closed without merging**